credits.txt

```diff
@@ -61,3 +84,187 @@ Beta Testers
   Sean Fahey
   Mauricio Gomes
   Iaz Poolar
+
+Special thanks to MCOfficer for
+creating, managing, and hosting
+the nightly builds that make
```

Comment:
might wanna change that to continuous, since that is supposed to replace the nightlies.

Reply:
Hmmm... I'm inclined to leave it like this until the continuous stuff is fully implemented and we're confident in it. But yeah, definitely something to change once we've made the shift. (yes, I know it *looks* up and running, but it is early yet)

Reply:
As far as i'm concerned, the continuous builds are already more stable than the nightlies could ever be.
The OSX version has "proper" library paths, and the windows version is built on windows using the official makefile, not some half-assed substitute. Plus there are 2 linux versions which the nightlies didn't have to begin with.

Reply:
And yet, your server's only ever been down once when I needed it - and GitHub seems to fail at stuff on a regular basis...

That being said, I changed it to say "nightly & continuous" in front of builds.